react/src/components/lablupTalkativotUI/EndpointLLMChatCard.tsx

```diff
@@ -130,6 +149,23 @@ const EndpointLLMChatCard: React.FC<EndpointLLMChatCardProps> = ({
           />
         )
       }
+      inputMessage={isSynchronous ? synchronizedMessage : undefined}
+      onInputChange={(v) => {
+        if (!_.isUndefined(v)) {
+          setSynchronizedMessage(v);
+        }
+      }}
+      submitKey={
+        chatSubmitKeyInfo?.id === submitId ? undefined : chatSubmitKeyInfo?.key
+      }
+      onSubmitChange={() => {
+        if (isSynchronous) {
+          setChatSubmitKeyInfo({
```

Comment:
```suggestion
      onSubmitChange={() => {
        setSynchronizedMessage('');
        if (isSynchronous) {
          setChatSubmitKeyInfo({
```

This fixes the problem of getting the previous value when clicking sync switch or when sending a message.